x-pack/plugins/discover_enhanced/ui_tests/tests/saved_searches.spec.ts

```diff
@@ -31,98 +31,93 @@ const assertDataViewIsSelected = async (page: ExtendedScoutTestFixtures['page'],
     'Incorrect data view is selected'
   ).toHaveText(name);
 
-test.describe(
-  'Discover app - saved searches',
-  { tag: ['@ess', '@svlSecurity', '@svlOblt', '@svlSearch'] },
-  () => {
-    const START_TIME = '2019-04-27T23:56:51.374Z';
-    const END_TIME = '2019-08-23T16:18:51.821Z';
-    const PANEL_NAME = 'Ecommerce Data';
-    const SEARCH_QUERY = 'customer_gender:MALE';
-    const SAVED_SEARCH_NAME = 'test-unselect-saved-search';
-    const filterFieldAndValue = {
-      field: 'category',
-      value: `Men's Shoes`,
-    };
-
-    test.beforeAll(async ({ esArchiver, kbnClient, uiSettings }) => {
-      await esArchiver.loadIfNeeded(testData.ES_ARCHIVES.ECOMMERCE);
-      await kbnClient.importExport.load(testData.KBN_ARCHIVES.DISCOVER);
-      await kbnClient.importExport.load(testData.KBN_ARCHIVES.ECOMMERCE);
-      await uiSettings.set({
-        defaultIndex: testData.DATA_VIEW_ID.ECOMMERCE,
-        'timepicker:timeDefaults': `{ "from": "${START_TIME}", "to": "${END_TIME}"}`,
-      });
+test.describe('Discover app - saved searches', { tag: ['@ess', '@svlSearch', '@svlOblt'] }, () => {
+  // TODO: Update to use an ES archive with an index accessible to 'viewer'
+  // for running this test against the Security serverless project.
+  const START_TIME = '2019-04-27T23:56:51.374Z';
+  const END_TIME = '2019-08-23T16:18:51.821Z';
+  const PANEL_NAME = 'Ecommerce Data';
+  const SEARCH_QUERY = 'customer_gender:MALE';
+  const SAVED_SEARCH_NAME = 'test-unselect-saved-search';
+  const filterFieldAndValue = {
+    field: 'category',
+    value: `Men's Shoes`,
+  };
+
+  test.beforeAll(async ({ esArchiver, kbnClient, uiSettings }) => {
+    await esArchiver.loadIfNeeded(testData.ES_ARCHIVES.ECOMMERCE);
```

Comment:
This test will work for Security project only with `admin` role, that has access to all indexes. It probably means generating new saved objects, leaving for later